projects/fuzzing-ethereum-network.md

```diff
@@ -0,0 +1,107 @@
+# Fuzzing the Ethereum network
+
+Create fuzzers for the ethereum network in order to find potential vulnerabilities .
+
+## Motivation
+
+In order to ensure the good running of Ethereum, it is imperative that is remains highly secure. As a large blockchain, Ethereum represents a target for potential hackers. 
+
+The Ethereum network and the differents protocols of the devp2p, plays an essential role in enabling communication between nodes, an essential element for the operation of the protocol. 
+
+Vulnerabilities or errors in this network could cause significant problems. It is therefore crucial to guarantee the maximum security of the network and exchanges between nodes.
+
+## Project description
+
+To increase security, I propose the creation of fuzzers, with the aim of potentially spotting vulnerabilities and bugs present on the Ethereum network and clients implementing the devp2p.
+
+The project was originally proposed by [Fredrik](https://github.com/fredriksvantes).
+
+The fuzzers will allow arbitrary/specific data to be sent to the various ethereum nodes through its network to cause unexpected behavior such as crashes, memory leaks and others...
+
+These behaviors will then be reported in order to be able to correct them and secure the network.
```

Comment:
This can be used for testing network upgrades and hardening p2p implementations in clients.